spec/controllers/idv/otp_verification_controller_spec.rb

```diff
@@ -89,6 +89,59 @@
       end
     end
 
+    context 'the user is going through in person proofing' do
+      before(:each) do
+        create(:in_person_enrollment, :establishing, user: user)
+      end
+
+      context 'the user uses sms otp' do
+        it 'doesn not save the phone number if the feature flag is off' do
+          put :update, params: otp_code_param
+
+          phone_config = user.establishing_in_person_enrollment&.notification_phone_configuration
+          expect(phone_config).to_not be_present
+        end
+
+        it 'saves the sms notification number to the enrollment' do
+          expect(IdentityConfig.store).to receive(:in_person_send_proofing_notifications_enabled).
```

Comment:
```suggestion
          allow(IdentityConfig.store).to receive(:in_person_send_proofing_notifications_enabled).
```

Reply:
I liked expect here because the feature should be checking the feature flag before saving the phone number. Any particular reason to use allow instead?

Also, once the feature flag is removed this line will start failing which will be a good prompt to remove it from the test (as then it would be unnecessary)

Reply:
Going to go ahead and merge as-is but I'll address this in a follow-up PR if you like